src/main/zip/plugin.xml

```diff
@@ -19,8 +19,8 @@
       Send a notification to a Slack channel.
 	</description>
     <properties>
-      <property name="webhook" required="true">
-        <property-ui description="The Incoming WebHook URL that will accept external messages into Slack." label="Incoming WebHook" type="textBox"/>
+      <property name="oauthAccessToken" required="true">
```

Comment:
So we don't want to get rid of the webhook capabilities here.  We still want that ability.

Please create a new plugin step that be called "Post Notification to Slack (With App Token)" or something like that.  Then it would resemble what we have in the internal Slack repository.

We have to be careful in every change that we make that the existing functionality still works for people.